clc/modules/object-storage-common/src/main/java/com/eucalyptus/objectstorage/msgs/ObjectStorage.groovy

```diff
@@ -66,8 +69,18 @@ import edu.ucsb.eucalyptus.msgs.StreamedBaseMessage
 @ComponentMessage(ObjectStorage.class)
 public class ObjectStorageResponseType extends ObjectStorageRequestType {
   HttpResponseStatus status; //Most should be 200-ok, but for deletes etc it may be 204-No Content
+  protected String bucketUuid;
 
   def ObjectStorageResponseType() {}
+
+  public String getBucketUuid() {
+    return bucketUuid;
+  }
+
+  public void setBucketUuid(String bucketUuid) {
+    this.bucketUuid = bucketUuid;
+  }
+
```

Comment:
No need to define getters and setters for bucketUuid in groovy class ObjectStorageResponseType. Groovy makes it happen automagically